qiskit/providers/ibmq/job/utils.py

```diff
@@ -63,3 +84,24 @@ def build_error_report(results: List[Dict[str, Any]]) -> str:
 
     error_report = 'The following experiments failed:\n{}'.format('\n'.join(error_list))
     return error_report
+
+
+def api_status_to_job_status(api_status: ApiJobStatus) -> JobStatus:
+    """Return the corresponding job status for the input API job status.
+
+    Args:
+        api_status (ApiJobStatus): API job status
+
+    Returns:
+        JobStatus: job status
+    """
+    return API_TO_JOB_STATUS[api_status]
+
+
+@contextmanager
+def api_to_job_error():
```

Comment:
The function is missing a return type annotation. Since nothing is returned, `None` should do.